Change default camera controls

diff -Naur betaflight-3.2.2/src/main/fc/rc_controls.c betaflight-3.2.2-patched/src/main/fc/rc_controls.c
--- betaflight-3.2.2/src/main/fc/rc_controls.c	2017-11-10 16:08:19.000000000 -0800
+++ betaflight-3.2.2-patched/src/main/fc/rc_controls.c	2017-11-13 19:14:49.304358576 -0800
@@ -323,22 +323,22 @@
 #endif
 
 #ifdef USE_CAMERA_CONTROL
-    if (rcSticks == THR_CE + YAW_HI + PIT_CE + ROL_CE) {
+    if (rcSticks == ##cameraControl.enter##) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_ENTER, 0);
         repeatAfter(3 * STICK_DELAY_MS);
-    } else if (rcSticks == THR_CE + YAW_CE + PIT_CE + ROL_LO) {
+    } else if (rcSticks == ##cameraControl.left##) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_LEFT, 0);
         repeatAfter(3 * STICK_DELAY_MS);
-    } else if (rcSticks == THR_CE + YAW_CE + PIT_HI + ROL_CE) {
+    } else if (rcSticks == ##cameraControl.up##) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_UP, 0);
         repeatAfter(3 * STICK_DELAY_MS);
-    } else if (rcSticks == THR_CE + YAW_CE + PIT_CE + ROL_HI) {
+    } else if (rcSticks == ##cameraControl.right##) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_RIGHT, 0);
         repeatAfter(3 * STICK_DELAY_MS);
-    } else if (rcSticks == THR_CE + YAW_CE + PIT_LO + ROL_CE) {
+    } else if (rcSticks == ##cameraControl.down##) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_DOWN, 0);
         repeatAfter(3 * STICK_DELAY_MS);
-    } else if (rcSticks == THR_LO + YAW_CE + PIT_HI + ROL_CE) {
+    } else if (rcSticks == ##cameraControl.upLong##) {
         cameraControlKeyPress(CAMERA_CONTROL_KEY_UP, 2000);
     }
 #endif
